dev-util/panopticon/files/qtpanopticon.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
Isn't this the same as panopticon.desktop?

Reply:
Except for the `qt` part in `/usr/bin/qtpanopticon` - someone changed that upstream :D

If you'd like to have it in https://github.com/das-labor/panopticon, we can also add it there :)

Reply:
That would be great. Please put it in `pkg/`